src/components/secrets/Secret.tsx

```diff
@@ -34,31 +36,227 @@ import { ReactComponent as Trash } from '../../assets/icons/ic-delete.svg'
 import { KeyValueFileInput } from '../util/KeyValueFileInput'
 import '../configMaps/ConfigMap.scss'
 import { decode } from '../../util/Util'
+import { ReactComponent as InfoIcon } from '../../assets/icons/ic-info-i-outline.svg'
+import { getCustomOptionSelectionStyle } from '../v2/common/ReactSelect.utils'
 
-let sampleJSON = [
-    {
-        key: 'service/credentials',
-        name: 'secret-key',
-        property: 'property-name',
-        isBinary: true,
+export const sampleJSONs = {
```

Comment:
Can we extract this sample JSON out to some separate file?

Reply:
Done